internal/utility/purl/composer_extractor.go

```diff
@@ -0,0 +1,21 @@
+package purl
+
+import (
+	"log"
+	"strings"
+
+	"github.com/google/osv-scanner/pkg/models"
+)
+
+func ExtractPURLFromComposer(packageInfo models.PackageInfo) (namespace string, name string, ok bool) {
+	nameParts := strings.Split(packageInfo.Name, "/")
+	if len(nameParts) != 2 {
+		log.Printf("invalid packagist package_name=%s", packageInfo.Name)
```

Comment:
can we return an error instead for all of these? we can replace the `ok bool` with an `err error`.